.github/ISSUE_TEMPLATE/Bug report form.yml

```diff
@@ -0,0 +1,127 @@
+- id: what-happened
+  type: textarea
+  attributes:
+    description: Please tell us what's not working as expected. Please include callstacks and error messages if possible.
+    label: What happened?
+    placeholder: Please describe the issue you encountered. Callstacks and error messages are helpful.
+  validations:
+    required: true
+- id: importance
+  type: dropdown
+  attributes:
+     label: Importance
+     description: May you help us understand the level of importance of this ask to you?
+     multiple: false
+     options:
+       - ----Please select----
+       - Blocking. My app's basic functions cannot work.
+       - Important. My app's user experience is significantly compromised.
+       - Moderate. This issue has a certain impact on my app. 
+  validations:
+      required: true
+- id: runtime-channel
+  type: dropdown
+  attributes:
+     label: Runtime Channel
+     description: Are you seeing this issue in stable release or prerelease of WebView2 runtime?
+     multiple: true
+     options:
+       - Stable release 
+       - Prerelease (Canary/Dev/Beta)      
+  validations:
+      required: true
+- type: markdown
+  attributes:
+    value: |
+        **Tip:** it's recommended to test your web content in the WebView2 control against Microsoft Edge Insider (preview) Channels (Beta, Dev, or Canary). [Download here](https://www.microsoft.com/en-us/edge/download/insider?form=MA13FJ). For more information please checkout the [development best practices](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/developer-guide#run-compatibility-tests-regularly-when-using-the-evergreen-runtime).
+- id: runtime-version
+  type: input
+  attributes:
+    description: Please enter the Runtime version you are using. (e.g. 114.0.1823.79)
+    label: Runtime Version
+  validations:
+    required: true
+- id: sdk-version
+  type: input
+  attributes:
+    description: Please enter the SDK version you are using. (e.g. 1.0.1905-prerelease)
+    label: SDK Version
+  validations:
+    required: true
+
+- id: framework
+  type: dropdown
+  attributes:
+    description: What framework are you using?
+    label: Framework
+    options:
+    - ----Please select----
+    - WPF
+    - Win32
+    - Winforms
+    - UWP/WinUI2
+    - WinUI3
+    - Unity Plugin
+    - Other
+  validations:
+    required: true
+- id: os
+  type: dropdown
+  attributes:
+    description: What operating system are you using?
+    label: Operating System
+    multiple: true
+    options:
+    - Windows 10
+    - Windows 11
+    - Earlier than Windows 10 (deprecated)
+    - Windows Server
+    - Xbox
+    - Hololens
+    - Other
+  validations:
+    required: true
+
+- id: os-version
+  type: input
+  attributes:
+    description: Please enter the OS version you are using. (e.g., 22H2, 21H1, etc.)
```

Comment:
Do customers typically know things like "22H2"? Or just build numbers?

Reply:
Good catch, I have seen developers using 22H2 but build numbers should make more sense. Updated this bit. We probably should add how to check OS version in the .md doc mentioned above?